site/src/pages/HealthPage/healthyColor.ts

```diff
@@ -0,0 +1,15 @@
+import { Theme } from "@mui/material/styles";
+
+export const healthyColor = (
+  theme: Theme,
+  healthy: boolean,
+  hasWarnings?: boolean,
```

Comment:
Just making sure: warnings could be for low-severity warnings or critical errors, right?

Reply:
healthy - it is working or not
warning - sometimes it is working but not as it should be

Reply:
Please align it with the JSON response. Severity is the major field now (instead of `healthy`). Warnings are reported if severity is elevated.

Reply:
Yep, in this case the JSON response should be the source of truth for severity.
If the API returns something that doesn't make sense, that should be fixed there and not in the FE.

Reply:
I'm now using severity!